functions.php

```diff
@@ -217,24 +217,68 @@ function ( $hidden ) {
  *
  * Add CSS classes to Gravity Forms fields.
  */
-add_filter( 'gform_field_css_class', 'custom_class', 10, 3 );
+add_filter( 'gform_field_css_class', 'gform_custom_field_class', 10, 3 );
 
```

Comment:
If was playing around this feature and develop a new solution to add that also disable the confirmation button and avoid confusions by the editors.

```
add_action( 'admin_init', function()  {
  echo '<style>
   .gf-link-disabled {
	opacity: 0.3;
	pointer-events: none;
    }
  </style>';
  echo '<script>
	addEventListener("load", () => {
	   const p4_gf_confirmation_select = document.getElementById("p4_gf_confirmation");
	   const p4_gf_confirmation_links = document.querySelectorAll("[href*=\'&subview=confirmation\']");

	   const _onChange = (target) => {
		if(target.value === "custom") {
		    p4_gf_confirmation_links.forEach(node => node.classList.add("gf-link-disabled"));
		} else {
		   p4_gf_confirmation_links.forEach(node => node.classList.remove("gf-link-disabled"));
		}
	}

	p4_gf_confirmation_select.addEventListener("change", evt => {
	   _onChange(evt.currentTarget);
	});

	_onChange(p4_gf_confirmation_select);
});
</script>';
});
```

Reply:
I'm not sure whether it's a good idea to change the core functionality too much 🤔 Also, you can still access this `Confirmations` menu via the forms list:

<img width="478" alt="Screenshot 2022-08-31 at 13 02 04" src="https://user-images.githubusercontent.com/6949075/187664148-cc01802c-ca8a-4ace-881f-e9e1a1ef9f7a.png">

Reply:
Yeah I agree. Since there is not a more elegant way of disabling this everywhere we can just leave it for now and rely on the tooltip explanation. We can also make sure this is covered in the handbook documentation.